src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -689,6 +698,11 @@ private Loan changeDueDate(DateTime dueDate, Loan loan) {
     return Collections.emptyList();
   }
 
+  @Override
+  public boolean isUnknown() {
```

Comment:
Why is this needed, as opposed to using polymorphism?

Reply:
This method is no longer relevant, so I removed completely.